### PR TITLE
Adds redirects EPs

### PR DESCRIPTION
**Pull request by d1am0nd**

### CODE REVIEW

File: src/endpoints/redirects/redirects.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+export function createPostRedirectEndpointRequest(
+  url: string,
+): BapiCall<Redirect> {
+  return {
+    method: 'POST',
+    endpoint: 'redirects',
+    params: {
+      service: 'go',
```
Comment (HenriBeck) [inline src/endpoints/redirects/redirects.ts]: @d1am0nd I would not add this here, we can just do this in our proxy

File: src/helpers/BapiClient.ts [hunk 2]
```diff
@@ -698,6 +703,24 @@ export class BapiClient {
       ),
   };
 
+  public readonly redirects = {
+    get: (params: GetRedirectsEndpointParameters = {}) =>
+      this.execute(createGetRedirectsEndpointRequest(params)),
+
+    post: (url: string) => {
```
Comment (HenriBeck) [inline src/helpers/BapiClient.ts]: Should we maybe name this better than `POST`, what about `match`?

**Review by HenriBeck — CHANGES_REQUESTED** [review]

File: src/endpoints/redirects/redirects.ts (hunk 1 above)
Comment (d1am0nd) [reply src/endpoints/redirects/redirects.ts]: It's a draft :D 

Needed it to test stuff

File: src/helpers/BapiClient.ts (hunk 2 above)
Comment (d1am0nd) [reply src/helpers/BapiClient.ts]: Yup, sounds good

File: src/endpoints/redirects/redirects.ts (hunk 1 above)
Comment (d1am0nd) [reply src/endpoints/redirects/redirects.ts]: Removed

File: src/helpers/BapiClient.ts (hunk 2 above)
Comment (d1am0nd) [reply src/helpers/BapiClient.ts]: Renamed

**Review by HenriBeck — APPROVED** [review]

**Pull request merged**